### PR TITLE
OKTA-581673 Updates to EL Manager Functions

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added a bullet point to address limitations.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-581673](https://oktainc.atlassian.net/browse/OKTA-581673)

### Netlify

* https://6400ebc038e261126033b2c5--reverent-murdock-829d24.netlify.app/docs/reference/okta-expression-language/#manager-assistant-functions

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md](https://github.com/okta/okta-developer-docs/blob/3f5d2fb660d24f57d2ca2d6b7b6388e6f2a4f7a7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language%2Findex.md) | 81 | 134 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/33c3a818-077a-4fc0-9c98-81f1907a5774) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.f6a30e57-69d7-43e4-8a3b-64bc9cea47fe?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 1]
```diff
@@ -109,13 +109,12 @@ Okta offers a variety of functions to manipulate attributes or properties to gen
...
 
-
-The following <ApiLifecycle access="deprecated" /> functions perform some of the same tasks as the ones in the above table.
+The following <ApiLifecycle access="deprecated" /> functions perform some of the same tasks as the ones in the previous table.
 
 | Function                          | Example                             | Input         | Output    |
 | --------                          | ---------                           | -------       | --------  |
 | `toUpperCase(string)`             | `toUpperCase(source.firstName)`     | Alexander     | ALEXANDER |
-| `toLowerCase(string)`             | `toLowerCase(source.firstName)`     | AlexANDER     | alexander |
+| `toLowerCase(string)`             | `toLowerCase(source.firstName)`     | AlexANDER     | oleander |
```
Comment (talbenari-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:117]: Was this change intentional?

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 2]
```diff
@@ -239,9 +237,10 @@ Okta supports the use of the time zone IDs and aliases listed in [the Time Zone
 The following should be noted about these functions:
 
 * Be sure to pass the correct App name for the `managerSource`, `assistantSource`, and `attributeSource` parameters.
-* At this time, `active_directory` is the only supported value for `managerSource` and `assistantSource`.
+* Currently, `active_directory` is the only supported value for `managerSource` and `assistantSource`.
 * Calling the `getManagerUser("active_directory")` function doesn't trigger a user profile update after the manager is changed.
 * The manager and assistant functions aren't supported for user profiles sourced from multiple Active Directory instances.
+* The manager and assistant functions aren't supported for user profile attributes from multiple app instances. That is, the expression `getManagerUser("active_directory", "google").firstName` returns null if your org has two or more instances of a `google` app.
```
Comment (talbenari-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:243]: 👍

**Review by talbenari-okta — APPROVED** [review]
One minor issue, otherwise LGTM (assuming grammatical changes have been approved by others)

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:117]: No. Thanks for the catch. Inadvertent typo.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md](https://github.com/okta/okta-developer-docs/blob/abc4d3441352f9ecc3c3618af8342a497e066651/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language%2Findex.md) | 81 | 139 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fde8c909-d6d2-4eee-ac33-732182b5ae6d) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.b9004800-8643-4cf1-81e5-0f28770a9728?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by brentschaus-okta — APPROVED** [review]
LGTM
